code/script.js

```diff
@@ -222,104 +226,195 @@ const generateBoard = () => {
   })
 }
 
+
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+ secret  = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
+
 // This function to start (and restart) the game
 const start = () => {
+  document.getElementById("winOrLose").style.display = "none";
+
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+
+  // The generateBoard function is called that draws the gameboard
+  generateBoard()
+  
+  // setSecret selects a secret character randomly from the CHARACTERS array
+  setSecret() 
+
+  // setting the currentQuestion object when you select something in the dropdown
+  selectQuestion()
 }
 
 // setting the currentQuestion object when you select something in the dropdown
```

Comment:
I would suggest on trying to keep the same line spaces between everything. It makes the code a bit easier to read and looks very nice aswell!

Reply:
Thank you for pointing this out! I usually make extra line spaces as I think its easier to read that way, but you are totally right that it's not good practice in a finished project.